### PR TITLE
EPMLABSBRN-213 add api for 3 series

### PR DESCRIPTION
**Pull request by ElenaSpb**
EPMLABSBRN-213 add api for 3 series

### CODE REVIEW

File: src/main/resources/initFiles/tasks_for_sentence_series.csv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+orderNumber words answerAudioFile answerParts
+1; (),(),(девочка, дедушка, бабушка), (бросает, читает, рисует),(),(); series3/девочка_читает.mp3; (девочка, читает)
```
Comment (goopnigoop) [inline src/main/resources/initFiles/tasks_for_sentence_series.csv]: Are all these spaces necessary here? Maybe better:
1;(),(),(девочка, дедушка, бабушка),(бросает, читает, рисует),(),();series3/девочка_читает.mp3;(девочка, читает)
CSV parsing should have rules and you have to include spaces to the rules

File: src/main/kotlin/com/epam/brn/model/Task.kt [hunk 2]
```diff
@@ -40,7 +41,14 @@ data class Task(
         joinColumns = [JoinColumn(name = "task_id", referencedColumnName = "id")],
         inverseJoinColumns = [JoinColumn(name = "resource_id", referencedColumnName = "id")]
     )
-    var answerOptions: MutableSet<Resource> = hashSetOf()
+    var answerOptions: MutableSet<Resource> = hashSetOf(),
+    @ManyToMany(cascade = [(CascadeType.ALL)])
```
Comment (goopnigoop) [inline src/main/kotlin/com/epam/brn/model/Task.kt]: We have to use only CascadeType.MERGE

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/model/Task.kt]: why

File: src/main/resources/initFiles/tasks_for_sentence_series.csv (hunk 1 above)
Comment (ElenaSpb) [reply src/main/resources/initFiles/tasks_for_sentence_series.csv]: с пробелами более читабельно

File: src/main/kotlin/com/epam/brn/model/Task.kt (hunk 2 above)
Comment (goopnigoop) [reply src/main/kotlin/com/epam/brn/model/Task.kt]: I would say just believe, you can read articles, main idea Tasks are independent of Resources when you dele task you shouldn't delete resources

Comment (ElenaSpb) [reply src/main/kotlin/com/epam/brn/model/Task.kt]: yes, thanks, fixed

**Pull request merged**